tools/rotate-azure-app-secret.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# This script takes in a new Azure application secret and applies it to all environments
+# To generate a secret:
+# - Log into the Azure portal (https://portal.azure.com/)
+# - Go to the DSP Terra Dev tenant
+# - Go to the App Registration service
+# - Click on the TDR application (https://portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationMenuBlade/Credentials/appId/22cb243c-f1a5-43d8-8f12-6566bcce6542/isMSAApp/)
```

Comment:
I cannot do this as I don't have permissions.

Reply:
well, that's not good.  One sec

Reply:
Huzzah I can create a secret now!